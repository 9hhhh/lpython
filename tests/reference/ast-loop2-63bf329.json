{
    "basename": "ast-loop2-63bf329",
    "cmd": "lpython --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/loop2.py",
    "infile_hash": "ca62d52ec7887e3a455cec87a0e49102e94e96db925f6c37edb792e7",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-loop2-63bf329.stdout",
    "stdout_hash": "946697009bcccd7ecbec9f12306b187e3cf9ee31fb9d86f2ad33b43c",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}