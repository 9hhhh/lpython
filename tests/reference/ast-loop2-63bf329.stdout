(Module [(FunctionDef test_for ([] [] [] [] [] [] []) [(AnnAssign (Name i Store) (Name i32 Load) () 1) (For (Name i Store) (Call (Name range Load) [(ConstantInt 0 ()) (ConstantInt 10 ())] []) [(If (Compare (Name i Load) Eq [(ConstantInt 0 ())]) [(Continue)] []) (If (Compare (Name i Load) Gt [(ConstantInt 5 ())]) [(Break)] [])] [] ())] [] () ())] [])
